Khan/wonder-blocks
# Configure dependabot so that netlify doesn't run a build for those PRs

## Summary:

See https://docs.netlify.com/site-deploys/manage-deploys/#skip-a-deploy for details.

Issue: none

## Test plan:

- land this PR
- re-enable netlify builds on PRs
- close current dependabot PRs so that it generates some new ones
- see that netlify doesn't run on the new PRs
